uqcsbot/__init__.py

```diff
@@ -73,30 +78,20 @@ def get_test_bot_token():
 
     json_contents = json.loads(response.content)
     if not json_contents['ok']:
-        logger.error('{0}'.format(json_contents['error']))
+        logger.error(json_contents['error'])
         sys.exit(1)
 
     for user_id in json_contents['members']:
-        if is_available_bot(user_id):
-            return {'id': user_id, 'token': BOT_TOKENS.get(user_id, None)}
+        info = get_bot_info(user_id)
+        if is_bot_active(info) and is_bot_avaliable(info, user_id):
+            return {'info': info, 'token': BOT_TOKENS.get(user_id, None)}
```

Comment:
`users.info` returns the user_id, so we can probably just return the info here and get the BOT_TOKEN later. This avoids relying on an awkward dictionary